lighthouse-cli/test/smokehouse/test-definitions/tricky-metrics/expectations.js

```diff
@@ -26,6 +26,22 @@ module.exports = [
       },
     },
   },
+  {
```

Comment:
Is it suspicious that the above expectations didn't change after dropping any throttling? Were the bounds just really really loose? Should we switch to a `±` assertion for them?